contracts/interfaces/IWavsServiceHandler.sol

```diff
@@ -2,9 +2,19 @@
 pragma solidity ^0.8.0;
 
 interface IWavsServiceHandler {
+    struct SignatureData {
+        address[] operators;
+        bytes[] signatures;
+        uint32 referenceBlock;
+    }
+    struct Envelope {
+        bytes payload;
+        uint256 eventId;
```

Comment:
Let's make this `bytes20`.
For padding, I assume better to put it in front, and give another `bytes12` fields there, which can be called "unused" or "ordering" if you really want to start prepping for that feature